app/mailers/application_mailer.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-# Autogenerated by rails generate mailer
-class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
-  layout 'mailer'
-end
```

Comment:
All the mailer code has been removed, as the workers should auto-recover from database connection losses. They'll return any messages they are processing to the queue, and wait until the connection recovers. As a result, we no longer need to terminate them from within the process. (Monit and systemd will still handle cases where the process crashes, which would never have generated these emails anyway)